IFC4x3/Sections/Domain specific data schemas/Schemas/IfcPortsAndWaterways/QuantitySets/Qto_FacilityPartBaseQuantities/DocQuantitySet.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocQuantitySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Qto_FacilityPartBaseQuantities" UniqueId="7dd73393-b5c7-45f3-8732-39d63a3cae15" ApplicableType="IfcFacilityPart">
```

Comment:
how does this pset make sense if facility parts are not supposed to be physical objects?

Reply:
well its the same principle as Qto_BuildingBaseQuantities , and Qto_BuildingStoreyBaseQuantities, it is supposed to be a volumetric organisational subdivision, so it can (but not always) have some sort of footprint area, volume.

Reply:
though it shouldn't have a weight which i just noticed...

Reply:
Okay, Should have been more specific I guess.